### ISSUE TITLE
Curves with emission are rendered in black

### ISSUE THREAD

**Issue by jkrumbiegel** [GPUOpen-LibrariesAndSDKs/RadeonProRenderSDK#60]
I'm using RadeonProRender.jl in Julia, and can't get emission working on a curve object. I'm not sure if it's a bug in the Julia implementation, a limitation of the SDK, or something else. I can't post a minimal example because this is all WIP Julia code, but here's an image showing the result:

![grafik](https://user-images.githubusercontent.com/22495855/224562980-2f525979-cb5e-45e7-84f8-f01d8e5f4331.png)

This is a line segment primitive which the RPRMakie package transforms into RPR Curves, roughly like this:

```julia
curve = RPR.Curve(context, points, indices, radius, Vec2f.(0.0, LinRange(0, 1, nsegments)),
                      fill(1, nsegments))
    material = RPR.UberMaterial(matsys)
material.emission_color = to_color(:red)
material.emission_weight = 1000
# I tried to set the double-sided mode to true or false in case the curves had inward normals for some reason
# material.emission_mode = false
set!(curve, material.node)
```

**Comment by ARozghon** [comment]
Thank you for your request, a ticket for devs has already been created 
could you share the scene to do the correct issue reproduce?

**Comment by jkrumbiegel** [comment]
> could you share the scene to do the correct issue reproduce?

Not really, as the Julia code was moving fast and it might not work anymore (plus your devs probably don't want to deal with Julia). But as far as I could see, any curves object with emission didn't work, so one should be able to construct a minimal example using that info :)

**Comment by ARozghon** [comment]
thank you, for your response. will check it issues

**Comment by goulf-3m** [comment]
With the same material and color, meshes can be red, but curves are always black. Any progress?
